src/java.ts

```diff
@@ -88,8 +88,7 @@ export async function serverSupportsJREProvisioning(
   properties: ScannerProperties,
 ): Promise<boolean> {
   if (properties[ScannerProperty.SonarScannerInternalIsSonarCloud] === 'true') {
-    //TODO: return to true once SC has the new provisioning mechanism in place
-    return false;
```

Comment:
@vdiez can you clarify that we need to wait for SC to be deployed to prod before we will have a scan in our branch?
https://cirrus-ci.com/task/4845465823346688?logs=sonarcloud_analysis#L25
```
[INFO]  Bootstrapper: Platform: linux x64
[DEBUG] Bootstrapper: Detecting proxy: No proxy detected
[INFO]  Bootstrapper: Server URL: https://sonarcloud.io
[INFO]  Bootstrapper: Version: 3.6.0
[DEBUG] Bootstrapper: Check if Server supports JRE Provisioning
[INFO]  Bootstrapper: JRE Provisioning is supported
[DEBUG] Bootstrapper: Detecting latest version of JRE
[DEBUG] Bootstrapper: Downloading JRE for linux x64 from /analysis/jres
[ERROR] Bootstrapper: An error occurred: Error: No JREs available for your platform linux x64
```

Reply:
nvm, SC is now in prod and the task runs successfully
https://sonarcloud.io/project/issues?id=SonarSource_sonar-scanner-npm&pullRequest=140&resolved=false&sinceLeakPeriod=true

but we have a few issues to address